include/ebpf_nethooks.h

```diff
@@ -78,7 +78,7 @@ bind_hook_t(bind_md_t* context);
 
 
 // Five-tuple Struct
-typedef struct five_tuple
+typedef struct _five_tuple
 {
     bool v4;
```

Comment:
does this compile with only stdint.h?

Reply:
There is <stdbool.h> as well and it does compile!